### PR TITLE
Add HttpTrigger-Java sample E2E test

### PR DESCRIPTION
**Pull request by pragnagopa**

### CODE REVIEW

File: WebJobs.Script.sln [hunk 1]
```diff
@@ -316,7 +316,7 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "HttpTrigger-Disabled", "Htt
 		sample\HttpTrigger-Disabled\index.js = sample\HttpTrigger-Disabled\index.js
 	EndProjectSection
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "TestFunctions", "test\TestFunctions\TestFunctions.csproj", "{5C308A72-5CF3-45E8-B64F-2C98F567054A}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "TestFunctions", "test\TestFunctions\TestFunctions.csproj", "{5C308A72-5CF3-45E8-B64F-2C98F567054A}"
```
Comment (mhoeger) [inline WebJobs.Script.sln]: Why is this changed? Mostly just curious :)

**Comment by pragnagopa** [comment]
@mhoeger - Added comments in Function.Java to generate jar file using maven
